File: beecrowd_urionlinejudge/uri1009SalarioBonus.cs
    using System;
    
    class URI {
    
        static void Main(string[] args) { 


String  NomeFuncionario;
double SalarioFixo,Vendas;


        NomeFuncionario=Console.ReadLine();
        SalarioFixo=float.Parse(Console.ReadLine());
        Vendas=float.Parse(Console.ReadLine());
    

            System.Console.WriteLine(String.Format( "TOTAL = R$ {0:0.00}",SalarioFixo+(Vendas*0.15)));
                
        }
    
    }

File: beecrowd_urionlinejudge/uri1009SalarioBonus.java
import java.util.Scanner;

import java.io.IOException;

public class Main {
 
    public static void main(String[] args) throws IOException {

string NomeFuncionario;
double SalarioFixo,Vendas;
    
Scanner input = new Scanner(System.in);

NumeroFuncionario = input.nextLine();
SalarioFixo = input.nextDouble();
Vendas = input.nextDouble();

System.out.printf("TOTAL = R$ %.2f\n",SalarioFixo+(Vendas*0.15));

    }
 
}


